codespell_lib/data/dictionary.txt

```diff
@@ -25278,6 +25278,7 @@ stoppping->stopping
 stopps->stops
 stopry->story
 storag->storage
+storaget->storage, target, targets,
```

Comment:
Do you have any examples of the latter corrections, they seem rather unlikely.

Reply:
Nope, I was only trying to preempt what you'd suggest ;)  Will remove.

Reply:
if i use https://grep.app/search?q=storaget&words=true, I only find c++ templates. Wouldn't it be better in the code dictionary?

Reply:
I found this typo in a code-comment (on an internal project) as "USB Mass Storaget Interface" which is why I suggested it here.
I've no idea what the "code dictionary" is, and I don't really have time to look into it right now, so I'll let you and @peternewman decide between you what to do.

Reply:
To cite Peter "So the code one (slightly confusingly) is where we put likely code typos, so you don't use that dictionary if you're checking code (if that makes sense)." from https://github.com/codespell-project/codespell/issues/1612

Reply:
> Nope, I was only trying to preempt what you'd suggest ;) Will remove.

Heh, I'm not that random am I @lurch ?

> I found this typo in a code-comment (on an internal project) as "USB Mass Storaget Interface" which is why I suggested it here.

Yeah that's great, I was only querying the "target" suggestions as I couldn't see how it would come up.

> if i use https://grep.app/search?q=storaget&words=true, I only find c++ templates. Wouldn't it be better in the code dictionary?

I see a few others dotted around, although the C++ template types do weight it quite heavily that way, so I guess it should go in there to avoid false positives. @sebweb3r do you fancy doing a PR based on this to move it if @lurch doesn't think he'll have time?

Reply:
I can hijack this MR, if @lurch is fine with it.

Reply:
Or he just puts the line into dictionary_code.txt himself and i add some additional finds. :-)

Reply:
Feel free to either add more commits to this PR, or create a new PR and close this one :+1: